packages/odata-annotation-core/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
```

Comment:
Same as in other package, remove duplicate options.

Reply:
done